src/Controller/AcademiesController.php

```diff
@@ -72,13 +74,46 @@ public function getAcademy(int $id)
             return new JsonResponse("There is no academy with id of: " . $id, Response::HTTP_NOT_FOUND);
         }
 
+        //Academies Info
+        $programsArray = array();
+        foreach ($academy->getPrograms() as $program) {
+            $programsArray[] = $program->getProgramName();
+        }
+
+        $priceArray = array();
+        foreach ($academy->getPrograms() as $price) {
+            $priceArray[] = $price->getProgramPrice();
+        }
+
+        $programs = new \stdClass();
+        $programs->academy_programs = $programsArray;
+        $programs->program_prices = $priceArray;
+
+        //Cities Info
+        $citiesArray = array();
+        foreach ($academy->getCities() as $city) {
+            $citiesArray[] = $city->getCity();
+        }
+
+        $addressArray = array();
+        foreach ($academy->getCities() as $address) {
+            $addressArray[] = $address->getCityAddress();
+        }
+
+        $cities = new \stdClass();
+        $cities->cities = $citiesArray;
+        $cities->addresses = $addressArray;
+
         return new JsonResponse(
             array(
                 'academy_id' => $academy->getId(),
                 'academy_name' => $academy->getAcademyName(),
                 'academy_email' => $academy->getAcademyEmail(),
                 'academy_url' => $academy->getAcademyUrl(),
                 'academy_logo' => $academy->getAcademyLogo(),
+                'academy_description' => $academy->getAcademyDescription(),
+                'academy_programs' => $programs,
+                'academy_cities' => $cities,
```

Comment:
From this response, how I will know in which city certain program is held?

Reply:
We will have filtering for that. As I understood, Kristina needs some info to show on the extended academy card, showing basic info: available programs, academy cities, prices, dates and reviews.

Reply:
Suggesting to give out json, which represents data structure similar to database:
`
{ "name": "academy_name", "email": "academy@email.com", "programs": [ { "name": "program1", "startDate": "2019-09-01", "endDate": "2020-02-01", "starRating": 4.6, "city": { "address": "some adress", "zip Code": "88654", "country": "netru" } } ] }
`